src/ERC20/ERC20/libraries/LibERC20.sol

```diff
@@ -1,38 +1,69 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.8.30;
 
+/// @title LibERC20 — Minimal ERC-20 Library for Diamond Storage Implementation
```

Comment:
Please remove "Minimal"  and "for Diamond Storage Implementation".      Every storage is a diamond storage implementation in this library.